initfiles/bin/init_thor

```diff
@@ -198,7 +199,7 @@ while [[ 1 ]]; do
         case $errcode in
         # TEC_Clean
         0)  log "Thormaster ($thorpid) Exited cleanly"
-            rm -f $instancedir/uslaves.start $PID_NAME $INIT_PID_NAME > /dev/null 2>&1
+            rm -f $PID_NAME $INIT_PID_NAME > /dev/null 2>&1
```

Comment:
I think we should leave in the remove file here since it was stopped clean and not restarted.